docs/database/database-metrics.md

```diff
@@ -35,6 +37,132 @@ and attributes but more may be added in the future.
 > until a transition plan to the (future) stable semantic conventions has been published.
 > Conventions include, but are not limited to, attributes, metric and span names, and unit of measure.
 
+## Database operation
+
+### Metric: `db.client.operation.duration`
+
+**Status**: [Experimental][DocumentStatus]
+
+This metric is [required][MetricRequired].
+
+When this metric is reported alongside a database operation span, the metric value SHOULD be the same as the database operation span duration.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[ 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 ]`.
+
+<!-- semconv metric.db.client.operation.duration(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `db.client.operation.duration` | Histogram | `s` | Duration of database client operations. |
+<!-- endsemconv -->
+
+<!-- semconv metric.db.client.operation.duration(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| [`db.instance.id`](../attributes-registry/db.md) | string | An identifier (address, unique name, or any other identifier) of the database instance that is executing queries or mutations on the current connection. This is useful in cases where the database is running in a clustered environment and the instrumentation is able to record the node executing the query. The client may obtain this value in databases like MySQL using queries like `select @@hostname`. | `mysql-e26b99z.example.com` | Recommended: If different from the `server.address` |
+| [`db.name`](../attributes-registry/db.md) | string | This attribute is used to report the name of the database being accessed. For commands that switch the database, this should be set to the target database (even if the command fails). [1] | `customers`; `main` | Conditionally Required: If applicable. |
```

Comment:
I think we need to normalize the case on all attributes used for metrics (i.e. `db.name` `Foo` should be `foo` everywhere).
otherwise we have cardinality problems on metrics and span names (or if we fix it for metrics only, we'll have non-correlated values between signals).

I guess if individual systems are case-sensitive, they can override it and not require lowercasing.

Totally fine to do it in a separate PR.

Reply:
Is there another example where we've had to solve this already, and if so where do we document this requirement?

I don't know if this is a good example, but [server.address](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/server.md) might be `example.com` or `Example.com`.

Reply:
One example is `http.request.method`, but I think it's not applicable here (see below). 

We've discussed it in the messaging semconv recently https://github.com/open-telemetry/semantic-conventions/issues/744 and decided not to normalize. Reasoning:
- in case of messaging things are usually case-sensitive, but individual extensions can override it and require normalization. Based on my 5 min research, mysql, postgresql and cassandra  are case-insensitive.
- but even if db/table name is case-insensitive, high cardinality would be really had to achieve - it's hard to imagine application querying against `foo` and `Foo` and `fOo` and all other possible combinations. Thing like this could only happen if app allows malicious users to control db/table name.

In case of HTTP method, user can send any method and it'd be added as a dimension on any request - so an easy attack. With everyone protecting from sql injection it seems unlikely for DB.

So I'm not sure normalization for DB name/ etc is really necessary.

Reply:
it looks like MySQL database and table names may be case sensitive on linux: https://stackoverflow.com/questions/6134006/are-table-names-in-mysql-case-sensitive

Reply:
it seems mysql is even more nuanced https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html 

There is a flag - [lower_case_table_names](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_lower_case_table_names) which controls it. 


Do you think the second argument still holds? it should be virtually impossible to create high-cardinality issues (unless app is also vulnerable to sql injection as well)

Reply:
essentially my proposal is: no guidance for generic case (meaning case-sensitive) , mysql extension can provide additional guidance and follow `lower_case_table_names` for specific attributes

Reply:
> Do you think the second argument still holds? it should be virtually impossible to create high-cardinality issues (unless app is vulnerable to sql injection as well)

👍

> essentially my proposal is: no guidance for generic case (meaning case-sensitive)

👍